userguide/tutorials/getting_started.adoc

```diff
@@ -136,48 +136,176 @@ You can also go to http://<IP>:8080/api.html to explore the KillBill APIs.
 
-==== Tomcat Installation Steps
+==== Tomcat Installation and configuration
 
-If you don't have it yet, you can download the Tomcat container http://tomcat.apache.org/download-80.cgi[here] (get the Core binary distribution). The documentation is available http://tomcat.apache.org/tomcat-8.0-doc/index.html[here].
+The first step in installing Killbill in Tomcat is to download and install Tomcat. For this, you need to follow the steps given below:
 
-In the rest of this guide, we will assume you have installed Tomcat in a directory called *CATALINA_HOME*.
+. Download the Tomcat container from http://tomcat.apache.org/download-80.cgi[here] (get the Core binary distribution). The downloaded zip file contains a folder called *apache-tomcat-8.5.59*
 
-Notes:
+. Extract *apache-tomcat-8.5.59* and place it on any location on your computer. This path will be refered to as *TOMCAT_HOME* from now on. (For example, if you place *apache-tomcat-8.5.59* in a directory called ```C:\Software```, *TOMCAT_HOME* refers to ```C:\Software\apache-tomcat-8.5.59```. It should look like this:
 
-* Unlike other installation methods, you will also need to install and configure your database manually. See https://docs.killbill.io/latest/userguide_deployment.html#_database_engine[our deployment guide] for more details.
-* Installation on Windows might work, but isn't tested nor recommended by the core team.
-* We recommend installing the https://tomcat.apache.org/native-doc/[Apache Tomcat Native Library] (automatically done for you when using Ansible, Docker, or in our Cloud Marketplace instances).
+ $TOMCAT_HOME
+     --bin
+     --conf
+     --lib
+     --logs
+     --temp
+     --webapps
+     --work
+ 
 
-==== Tomcat configuration
 
-Recommended Tomcat configuration files can be found https://github.com/killbill/killbill-cloud/tree/master/ansible/templates/tomcat/conf[here]. The `.j2` extension is for the Jinja templating language: you must render the files first using your custom variables, which depend on your environment (take a look at an example of such variables https://github.com/killbill/killbill-cloud/blob/master/ansible/group_vars/all.yml[here]). https://ansible.sivel.net/test/[This website] can be used to perform the rendering online if you do not have Jinja installed.
+. Ensure that you have Java 8 or higher installed
 
-Once rendered, `setenv2.sh` should be placed in *CATALINA_HOME/bin* and all other files should be placed in *CATALINA_HOME/conf*.
+. Set the *JAVA_HOME* environment variable to the path of your JDK installation (For example, if JDK is installed at `C:\Software\jdk1.8.0_102`, you need to set *JAVA_HOME* to `C:\Software\jdk1.8.0_102`)
```

Comment:
Can you also provide a link to download the JDK?

Reply:
Done, added link to download JDK